webapp2.py

```diff
@@ -251,9 +260,9 @@ def get_all(self, argument_name, default_value=None):
     def arguments(self):
         """Returns a list of the arguments provided in the query and/or POST.
 
-        The return value is a list of strings.
+        The return value is an ordered list of strings.
         """
-        return list(set(self.params.keys()))
+        return sorted(set(self.params.keys()))
```

Comment:
Oh, sorry my mistake. It's request arguments.
In python 2 this construct `list(set(self.params.keys()))` will return the same order as was in request before converting to set, in python 3 - nope. fixed with ordered dict